platform/ATParser.h

```diff
@@ -57,15 +57,17 @@ class ATParser
     // Parsing information
     const char *_output_delimiter;
     int _output_delim_size;
+    char _in_prev;
     bool _dbg_on;
     bool _aborted;
 
     struct oob {
```

Comment:
out of curiosity, what `oob` stands for? `oobs` might not be intuitive that it is linked list of `oob`